virocon/distributions.py

```diff
@@ -1347,3 +1347,158 @@ def _fit_mle(self, sample):
 
     def _fit_lsq(self, data, weights):
         raise NotImplementedError()
+ 
+class VonMisesDistribution(Distribution):
+    """
+    A Von Mises (Circular Norm) Distribution. 
+    
+    The distributions probability density function is given by [1]_: 
+    
+    :math:`f(x) = \\frac{\\exp{k \\cos{x - \\mu}}}{2 \\pi I_0(k) \\sigma}`
+     
+    The distribution is used to model wind-wave misalignment in [2]_. 
+    Being a circular norm distribution it can be used to model direction. 
+    
+    Parameters
+    ----------
+    mu : float
+        Location parameter, the mean.
+        Defaults to 0.
+    sigma : float
```

Comment:
If sigma always should be fixed to 1, we should not allow the user to change that, i.e remove sigma. Or is there a good reason to keep it?

Reply:
The description I wrote was misleading. I have noticed that the scipy has trouble fitting the distribution if sigma is not fixed (as suggested in the stackoverflow thread I linked). I have only tried the code succesfully with f_sigma = 1, leading to the incorrect description from my side. 

I would propose to change the default f_sigma from None to 1. This way the user is provided with a stable starting point, but can decide to change f_sigma if he/she wishes. 

Not sure if this is a clean solution that can be explained simply in the documentation, which in my view would be the main reason to remove the sigma parameter.

Reply:
The answer https://stackoverflow.com/a/39020834 in the stackoverflow post has the following comment: 
"The scale parameter doesn't make any sense for the von Mises distribution anyway. The distribution is on a circle. It can't be scaled. – Neil G Apr 15, 2021 at 21:02
Scipy uses the same parameter structure (shape(s), loc, scale) for all it's distributions. So for consinstency scipy exposes the scale parameter here, too. But if for the Von Mises distributions a scale parameter does not make sense, we should not pass it through to our API. So dropping sigma would be consistent with our approach, as e.g. our LogNormal distribution does not use a location paramater (in contrast to scipy's lognorm).

Reply:
Ok, thanks for the clarification. Dropped the sigma parameter and fixed it to 1 in a similar way to what was done in the LogNormal distribution